src/lane_follow/include/LineDetect.h

```diff
@@ -0,0 +1,194 @@
+/*------------------------------------------------------------------------------------------*\
+Lane Detection
+
+General idea and some code modified from:
+chapter 7 of Computer Vision Programming using the OpenCV Library.
+by Robert Laganiere, Packt Publishing, 2011.
+
+ This program is free software; permission is hereby granted to use, copy, modify,
+and distribute this source code, or portions thereof, for any purpose, without fee,
+subject to the restriction that the copyright notice may not be removed
+or altered from any source or altered source distribution.
+The software is released on an as-is basis and without any warranties of any kind.
+In particular, the software is not guaranteed to be fault-tolerant or free from failure.
+The author disclaims all warranties with regard to this software, any use,
+and any consequent failure, is purely the responsibility of the user.
+ 
+Copyright (C) 2013 Jason Dorweiler, www.transistor.io
+\*------------------------------------------------------------------------------------------*/
+
+/*
+ * Created by: Raad Khan
+ * Created On: April 23, 2017
+ * Description:
+ * Usage:
+ * Subscribes to:
+ * Publishes to:
+ *
+ */
+
+#ifndef LINEF
+#define LINEF

```

Comment:
Hmm, our standard in the `IGVC-2017` repo seems to be `PackageName_Node_Name_H` (in all uppercase). Although this has never been enforced, we may as well follow that here, so this would be `LANE_FOLLOW_LINE_DETECT_H`